app/static/playwright.config.ts

```diff
@@ -4,7 +4,8 @@ const config: PlaywrightTestConfig = {
     testMatch: '*.etest.ts',
     use: {
         baseURL: "http://localhost:3000",
-        screenshot: "only-on-failure"
+        screenshot: "only-on-failure",
+        actionTimeout: 0
```

Comment:
actionTimeout ensures the tests wait for click event.

Reply:
https://playwright.dev/docs/test-timeouts